docs/_docs/templates.md

```diff
@@ -178,7 +179,8 @@ you come up with your own tags via plugins.
     <tr>
       <td>
         <p class="name"><strong>Group By Expression</strong></p>
-        <p>Group an array's items using a Liquid expression.</p>
+        <p>Group an array's items using a Liquid expression.
+        {% include docs_version_badge.html version="3.4.0" %}</p>
```

Comment:
Note: We don't want to use `{%- -%}` syntax here, because we want a space after the period before the badge.